tools/onnx_subgraph/test_model_download.sh

```diff
@@ -0,0 +1,15 @@
+pip install onnx onnxsim
```

Comment:
IIUC, this will try to install to the system, not inside user path.
is this your intention?

why not use `venv` ?